static/app/views/explore/logs/types.tsx

```diff
@@ -25,13 +25,14 @@ export enum OurLogKnownFieldKey {
   CODE_LINE_NUMBER = 'tags[code.line.number,number]',
   CODE_FUNCTION_NAME = 'code.function.name',
 
+  RELEASE = 'release',
+  TEMPLATE = 'message.template',
+  PARENT_SPAN_ID = 'trace.parent_span_id',
+  SDK_NAME = 'sdk.name',
+  SDK_VERSION = 'sdk.version',
```

Comment:
```suggestion
  ENVIRONMENT = 'environment',
  RELEASE = 'release',
  ORIGIN = 'origin'
  TEMPLATE = 'message.template',
  PARENT_SPAN_ID = 'trace.parent_span_id',
  SDK_NAME = 'sdk.name',
  SDK_VERSION = 'sdk.version',
```

I think we need `origin` and `environment` here?

Reply:
these are only the ones that are used somewhere